assets/controller.yaml

```diff
@@ -20,6 +20,7 @@ spec:
       labels:
         app: azure-file-csi-driver-controller
       annotations:
+        openshift.io/required-scc: restricted-v2
```

Comment:
I thought "hostNetwork: true" requires "privileged". Am I wrong? How do you decide between "restricted-v2" and "privileged", and why this PR doesn't add similar annotation to node.yaml?